draft-ietf-quic-transport.md

```diff
@@ -5398,6 +5398,10 @@ and preferred_address transport parameter connection IDs.  The peer SHOULD
 retire the corresponding connection IDs and send the corresponding
 RETIRE_CONNECTION_ID frames in a timely manner.
 
+An endpoint that receives a NEW_CONNECTION_ID frame with a sequence number
+smaller than the Retire Prior To field of a NEW_CONNECTION_ID frame received
+before, MUST immediately send a corresponding RETIRE_CONNECTION_ID frame.
```

Comment:
```suggestion
frame MUST immediately send a corresponding RETIRE_CONNECTION_ID frame.
```

Reply:
I would potentially also add
```suggestion
MUST immediately send a corresponding RETIRE_CONNECTION_ID frame that 
retires the newly received connection ID.
```
Since there's two NCID frames being referenced here.

Reply:
Thanks! I applied your suggestions.